wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/DCMotorSim.java

```diff
@@ -243,6 +248,16 @@ public double getInputVoltage() {
     return getInput(0);
   }
 
+  /**
+   * Gets the input voltage for the DC motor.
+   *
+   * @return The DC motor's input voltage.
+   */
+  public Voltage getInputVoltageVolts() {
+    m_voltage.mut_setMagnitude(getInput(0));
```

Comment:
Ditto

Reply:
You can't have two duplicate functions with different return types, so that doesn't work.

Reply:
I don't mean changing the function name but using the call from getInputVoltage in here instead of getInput(0)